src/patternfly/components/Form/form.scss

```diff
@@ -25,7 +25,7 @@
 
   // actions
   --pf-c-form__actions--MarginBottom: calc(var(--pf-c-form__actions--child--MarginBottom) * -1);
-  --pf-c-form__actions--child--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-form__actions--child--MarginRight: var(--pf-global--spacer--md);
```

Comment:
is this meant to be in this PR?

Reply:
nope! weird, rebased and looks like it went away.

Reply:
nice